package.json

```diff
@@ -36,6 +36,7 @@
     "webpack-cli": "^5.1.1"
   },
   "dependencies": {
+    "@babel/core": "^7.23.9",
```

Comment:
Good idea to update dependencies as well. Running something like https://www.npmjs.com/package/npm-check-updates manually once done and committing module updates in a separate commit would be totally fine. But handling this will be part of repository maintenance any way.

Reply:
Alright, shall we just keep this as part of the PR then?

Reply:
This can be left out, yes.